packages/flutter_tools/lib/src/update_packages_pins.dart

```diff
@@ -28,6 +28,7 @@ const Map<String, String> kManuallyPinnedDependencies = <String, String>{
   'leak_tracker': '10.0.7', // https://github.com/flutter/devtools/issues/3951
   'leak_tracker_testing': '3.0.1', // https://github.com/flutter/devtools/issues/3951
   'leak_tracker_flutter_testing': '3.0.8', // https://github.com/flutter/devtools/issues/3951
+  'path_provider_android': '2.2.1', // https://github.com/flutter/flutter/issues/140796
```

Comment:
FYI @goderbauer & @gmackall I re-pinned path_provider_android to unblock this roll :'(